### DISCUSSION TITLE
Limit GPU Wattage in LLTs custom power plan

### DISCUSSION THREAD

**Discussion by durstig66** [BartoszCichecki/LenovoLegionToolkit#945] [Ideas]
With the introduction of the custom power mode and the advanced setting to limit the adaptive +15 watts and then the further 10watts by default setting i was wondering if it would take a lot of work to implement a full power control slider for the GPU,
for example when im in bed playing on the built in screen i limit my gpu wattage (to keep noise down) by underclocking in afterburner, this is however not a very elegant solution.
An implementation of a 40-115 ´watt slider for the gpu in LLTs custom power plan would create the perfect all in one tool for anyone that has an interest in noise reduction or perhaps also power efficiency.
(cpu wattage can already be controlled pretty easily by windows power plan cpu limits)

**Comment by BartoszCichecki** [comment]
All the options that are provided by Lenovo are already exposed on the UI, so if you need more customization you need to use something else.

**Reply by durstig66** [reply]
I see, thanks for clearing things up, and thanks for your work.
